webrtc.html

```diff
@@ -3462,6 +3462,16 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             renegotiated.  After the codecs are renegotiated, they are
             set to the value negotiated, and <code>setParameters</code> needs to be
             called again to re-apply codec preferences.</p>
+
+            <p> The setParameters does not cause renegotiation at the SDP level
+            and can only be used to change what the media stack is sending or
+            receiving withint the envelope negotiated by the Offer/Answer. The
+            atributes in the <code>RTCRtpParameter</code> dictionary are
+            designed to not enable this so things like <code>ssrc</code> that
+            can not be changed are read only and things like bitrate are only
+            controllable with a <code>maxBitrate</code> which is merged with
+            bitrate constraints that may be signaled in SDP. </p>
```

Comment:
I think that "merged" needs to be better defined (or state that it's not merged, but applied separately as another limit on bitrate, separate to that imposed by the peer via SDP).